packages/lesswrong/components/users/UsersProfile.tsx

```diff
@@ -368,21 +391,20 @@ const UsersProfileFn = ({terms, slug, classes}: {
             <div className={classes.usernameTitle}>
               <div>{username}</div>
               {forumTypeSetting.get() === "EAForum" && currentUser?._id != user._id && (
-                <NewConversationButton user={user} currentUser={currentUser}>
-                  <Button color="primary" variant="contained" className={classes.messageBtn} data-cy="message">
-                    Message
-                  </Button>
-                </NewConversationButton>
+                <div className={classes.messageBtnDesktop}>{eaMessageBtn}</div>
               )}
             </div>
             {mapLocationNode}
+            {forumTypeSetting.get() === "EAForum" && currentUser?._id != user._id && (
+              <div className={classes.messageBtnMobile}>{eaMessageBtn}</div>
+            )}
             <Typography variant="body2" className={classes.userInfo}>
               { renderMeta() }
               { currentUser?.isAdmin &&
                 <div>
                   <DialogGroup
                     actions={[]}
-                    trigger={<span>Register RSS</span>}
+                    trigger={<span>Add RSS</span>}
```

Comment:
Changed this text so that this row of links won't wrap on desktop

Reply:
It's worse text, but worth it